README.md

```diff
@@ -37,19 +37,30 @@ your Oracle database and perform various database operations as needed.
 
 The package could be installed via composer:
 
-```php
+```shell
 composer require yiisoft/db-oracle
 ```
 
-## Usage 
+## Documentation
```

Comment:
Suggest move all text to language specific files and make this block like so:

![image](https://github.com/yiisoft/db-oracle/assets/525501/15fb67eb-80f7-4938-9e3e-762c2a5629e3)

Reply:
I planned to isolate the main README.md to avoid modifications. Guides, etc. would be in a readme.md inside the 'docs' folder.